tg-portfolio/components/skills/Experience.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const Experience = ({ companies }) => (
+  <div>
+    <div className="carousel rounded-lg w-full animate-in slide-in-from-bottom-48 duration-1000">
+      <div id="item1" className="carousel-item w-full bg-neutral/60 z-20">
+        <div className="flex flex-col gap-5 p-5">
+          <h1 className=" font-bold text-black z-30 test text-4xl ">{companies[0].Company}</h1>
+          <div>
+            <h2 className="font-semibold aboutTitle text-3xl text-black/75 ">{companies[0].title}</h2>
+          </div>
+          <div>
+            <p>{companies[0].experience}</p>
+          </div>
+        </div>
+      </div>
```

Comment:
It's the same structure throughout the page with a difference in changing data. Create a new component and pass the data accordingly:

```javascript
{companies.map((company) => (
   <ExperienceColumn 
      key={company.title}
      title={company.title} 
      company={company.Company}
      experience={company.experience}
   />
) )}
```

ExperienceColumn will contain the above repeating structure

Reply:
Thank-you 😄